docs/topics_5/web-openshift-troubleshoot.adoc

```diff
@@ -21,26 +21,26 @@ NOTE: If you have deployed the {WebName} by pasting in a JSON template, no image
 [id='examine_logs_{context}']
 == Examining and collecting {WebName} logs
```

Comment:
I suggest removing this heading and intro text. Then create new modules for downloading logs with the console and the CLI. You can reference both modules here.